rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -869,17 +873,20 @@ object RudderConfig extends Loggable {
   // apiv8 add policy mode in node API and settings API
   val apiV8 = nodeApi8 :: settingsApi8 :: apiV7.filter( _ != nodeApi6)
 
+  val apiV9 = ncfAPI :: apiV8
+
   val apis = {
     Map (
-        //Rudder 3.0
+        //Rudder 3.0D
         ( ApiVersion(5,true) -> apiV5 )
         //Rudder 3.1
       , ( ApiVersion(6,true) -> apiV6 )
```

Comment:
we forgot to remove that in 4.2, don't we?

Reply:
You done it in another PR